google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/RpcViewConstants.java

```diff
@@ -66,103 +70,124 @@ class RpcViewConstants {
                   4096.0, 8192.0, 16384.0, 32768.0, 65536.0, 131072.0, 262144.0, 524288.0,
                   1048576.0, 2097152.0)));
 
+  private static final List<TagKey> baseTags =
```

Comment:
And I think it would be better to inline these...they arent the same for all of the metrics so they arent quite base. IT would also make it easier to review :)